src/WebJobs.Script/Diagnostics/MetricEventNames.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.WebJobs.Script.Diagnostics
+{
+    public static class MetricEventNames
```

Comment:
Created a constants class for these so we can view the namespaces together in one place to ensure consistency. If the strings are instead littered all over the code, it is hard to achieve consistency.